psi4/driver/inputparser.py

```diff
@@ -796,7 +789,9 @@ def future_replace(m):
 
     # Move up the psi4.core namespace
     for func in dir(core):
-        temp = temp.replace("psi4." + func, "psi4.core." + func)
+        # Except fns that have valid wrappers in driver
+        if func not in ['set_memory']:
+            temp = temp.replace("psi4." + func, "psi4.core." + func)
```

Comment:
This could get rough, would you consider moving the `psi4.core.set_memory` to a different internal name?